### ISSUE TITLE
Message is too long

### ISSUE THREAD

**Issue by vimnewbie93** [roxma/vim-window-resize-easy#2]
I'm using GVim and discovered the problem, that if the window is too small for the message 

`"window resizing... horizontal smaller [<], horizontal greater [>], vertical decrease [-], vertical increase [+], others [_] [=] "`,

you get the message `"Press ENTER or type command to continue"`. This disables the timeout as it waits for the user to press a key.

I don't know how to fix this in a proper way, a workaround would be to shorten the message.

**Comment by vimnewbie93** [comment]
I have seen that you already shortened the message. I found a way to make this even more robust (although I can't imagine anyone having a window that small).

`echo 'window resizing... [ > < + - _ = ]'[0:&columns-13]`

This works for me, as it checks the current columns, subtracts 13 and prints only that part of the string. I found that it has to be 13 through trial and error. Maybe there is an even cleaner way to do this?
